generators/ci-cd/templates/jenkins/_jenkins.yml

```diff
@@ -0,0 +1,9 @@
+version: '2'
+services:
+  jenkins:
+    image: <%= DOCKER_JENKINS %>
+    ports:
+      - 49001:8080
+      - 50000:50000
+    volumes:
+      - ./jenkins_data:/var/jenkins_home
```

Comment:
To be consistent with all other compose files, I still think it should be:
```
# warning: be sure the local jenkins_home folder is created before using this volume
# - ~/volumes/jenkins_home:/var/jenkins_home
```

I will explain why:

The 1st time I launch `docker-compose -f src/main/docker/jenkins.yml up`. Here what I have:
```
$ docker-compose -f src/main/docker/jenkins.yml up
WARNING: Found orphan containers (docker_jhmysql-sonar_1) for this project. If you removed or renamed this service in your compose file, you can run this command with the --remove-orphans flag to clean it up.
Creating docker_jenkins_1
Attaching to docker_jenkins_1
jenkins_1  | touch: cannot touch ‘/var/jenkins_home/copy_reference_file.log’: Permission denied
jenkins_1  | Can not write to /var/jenkins_home/copy_reference_file.log. Wrong volume permissions?
docker_jenkins_1 exited with code 1
```
It is because the `src/main/docker/jenkins_data` is not created, so it will be created in local under `root` user

So I delete this folder. Then, if you created this folder before launching the container, it will work:
```
╭─pgrimaud@xps  ~/tmp/01-default-mysql  ‹master› 
╰─$ mkdir -p src/main/docker/jenkins_data
╭─pgrimaud@xps  ~/tmp/01-default-mysql  ‹master› 
╰─$ docker-compose -f src/main/docker/jenkins.yml up
WARNING: Found orphan containers (docker_jhmysql-sonar_1) for this project. If you removed or renamed this service in your compose file, you can run this command with the --remove-orphans flag to clean it up.
Recreating docker_jenkins_1
Attaching to docker_jenkins_1
jenkins_1  | Running from: /usr/share/jenkins/jenkins.war
jenkins_1  | webroot: EnvVars.masterEnvVars.get("JENKINS_HOME")
jenkins_1  | Mar 09, 2017 7:12:55 AM Main deleteWinstoneTempContents
jenkins_1  | WARNING: Failed to delete the temporary Winstone file /tmp/winstone/jenkins.war
jenkins_1  | Mar 09, 2017 7:12:55 AM org.eclipse.jetty.util.log.JavaUtilLog info
jenkins_1  | INFO: Logging initialized @514ms
(...)
```

But the `jenkins_data` is not in .gitignore so it can be accidentaly commited:
```
╭─pgrimaud@xps  ~/tmp/01-default-mysql  ‹master*› 
╰─$ gst
Sur la branche master
Fichiers non suivis:
  (utilisez "git add <fichier>..." pour inclure dans ce qui sera validé)

	src/main/docker/jenkins_data/
```